validation-test/Sema/type_checker_perf/fast/issue43386_array_of_ranges.swift.gyb

```diff
@@ -0,0 +1,16 @@
+// RUN: %scale-test --begin 1 --end 15 --step 1 --select NumLeafScopes %s -Xfrontend=-solver-expression-time-threshold=1
```

Comment:
Is this the appropriate metric?

Reply:
CC @xedin

Reply:
Sorry, I was referring to  `NumLeafScopes`.

> I'd suggest you actually form a single array with `0...1` and `0..<1` together

Will do, thanks!

Reply:
Done!